### PR TITLE
Added Dockerfile and docker-compose for kitty deployment

### PR DESCRIPTION
**Pull request by walter-bd**
I just added some files to add compatibility to docker; the changes are:

1.  Created  Dockerfile and docker-compose files to create the image and deploy it.
2.  Edited db.py to add an env variable to support simple volume creation.
3.  Edited README.md to list the commands to build and deploy kitty with docker compose.

### CODE REVIEW

**Comment by adamnwerbik** [comment]
like and subscribe for more

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.8.10
+RUN apt-get update -y
```
Comment (flukiluke) [inline Dockerfile]: Suggest combining the apt invocations to the same RUN with && to reduce the number of layers and clean up with `rm -rf /var/lib/apt/lists/*`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+RUN apt-get upgrade -y
+RUN apt-get install libfreetype6-dev libxft-dev fortune -y
```
Comment (flukiluke) [inline Dockerfile]: What are libfreetype and libxft used for? I was able to build & run okay without them.

Comment (adamnwerbik) [reply Dockerfile]: kinda suss ngl

stay safe king!

Comment (walter-bd) [reply Dockerfile]: Just some font libs that @olliequ installed in his version for font compatibility with Matplotlib. If you want, I can remove them.

Comment (olliequ) [reply Dockerfile]: They were recommended to be installed on my DO Ubuntu server by the guide I was using: 
> For some reason, Matplotlib and a few other packages have issues with Ubuntu, but you can solve these by running:
> sudo apt-get install libfreetype6-dev libxft-dev
> sudo apt-get install python-setuptools

The guide: https://towardsdatascience.com/lessons-learned-digital-ocean-for-python-3-e2442db4246f

File: Dockerfile (hunk 1 above)
Comment (walter-bd) [reply Dockerfile]: Agree; going to do this.

**Pull request closed without merging**